src/components/common/Actions.vue

```diff
@@ -115,8 +121,10 @@ export default class Actions extends Vue {
    */
   private async onClickSaveResume (): Promise<void> {
     this.setIsSavingResuming(true)
-    await this.sleep(1000)
+    const filing = await this.buildFiling()
+    await this.saveFiling(filing, true)
     this.setIsSavingResuming(false)
+    window.location.assign(this.authUrl)
```

Comment:
FYI, this will eventually be a redirect to a dashboard, but we don't know the URL for that yet so this is an OK assumption for now.
(no change required)